spec/spec_helper.rb

```diff
@@ -24,6 +24,23 @@
 end
 require 'rails_helper'
 
+OmniAuth.config.test_mode = true
```

Comment:
@michaeljevans @ejdelsztejn do we need this if it's already in michael's upcoming work? should we just merge and then overwrite later if not needed?